openapi/openapi.yaml

```
@@ -1056,6 +1056,10 @@ paths:
     $ref: ./paths/tracking@api.yaml
   '/tracking/api/{id}':
     $ref: './paths/tracking@api@{id}.yaml'
+  /tracking/taxjar:
+    $ref: ./paths/tracking@taxjar.yaml
+  '/tracking/taxjar/{id}':
+    $ref: './paths/tracking@taxjar@{id}.yaml'
```

I think these should change. Any tax integration log should be retrieved via a unified API.
```suggestion
  /tracking/taxjar:
    $ref: ./paths/tracking@taxes.yaml
  '/tracking/taxjar/{id}':
    $ref: './paths/tracking@taxes@{id}.yaml'
```